src/otx/v2/adapters/torch/mmengine/engine.py

```diff
@@ -334,17 +422,17 @@ def validate(
                 work_dir=str(target_folder),
                 experiment_name="otx_validate",
                 cfg=Config({}),  # To prevent unnecessary dumps.
-                **self.config,
+                **config,
             )
         elif update_check:
-            self.runner._val_dataloader = self.config["val_dataloader"]  # noqa: SLF001
-            self.runner._val_loop = self.config["val_cfg"]  # noqa: SLF001
-            self.runner._val_evaluator = self.config["val_evaluator"]  # noqa: SLF001
+            self.runner._val_dataloader = config["val_dataloader"]  # noqa: SLF001
+            self.runner._val_loop = config["val_cfg"]  # noqa: SLF001
+            self.runner._val_evaluator = config["val_evaluator"]  # noqa: SLF001
```

Comment:
Could you explain why should we set this value? It seems they should be set when runner is created

Reply:
This is for reusing the runner.  
If we reuse the runner, we only need to change those three variables in the runner to use the new val_dataloader.